shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -193,6 +193,9 @@ @implementation FlutterEngine {
 
   // Pointer to the Dart AOT snapshot and instruction data.
   _FlutterEngineAOTData* _aotData;
+
+  // FlutterMacOSGLCompositor is created by the engine.
+  std::unique_ptr<flutter::FlutterMacOSGLCompositor> _macOSCompositor;
```

Comment:
Add a note about when this is used.

Reply:
Done

Reply:
Make it clear when the compositor will be used to create backing stores as opposed to `OnFBO` being called. How does this fit in with `FlutterOpenGLRendererConfig` being passed?